syntax = "proto3";

package ExamplePrototypes;

message CoordinatesUpdate {
  int32 x = 1;
  int32 y = 2;
}

message CoordinatesGet {
  uint32 identifier = 1;
}